docs/guidance/map/buyers_suppliers.md

```diff
@@ -42,9 +42,25 @@ Siemens and Microsoft bid as a consortium for a contract to develop a new medica
 
 The contract is awarded to the consortium; however, the legal entity for the consortium is not created until after the contract award.
 
-Both Siemens and Microsoft are listed as suppliers on the contract award in OCDS, with the respective legal entity identifiers for each organization:
-
-```{csv-table-no-translate}
-:header-rows: 1
-:file: ../../examples/organizations/consortia_simple.csv
+Both Siemens and Microsoft are listed as suppliers on the contract award in OCDS, with the respective ids for each organization that match their id in the `Parties` array:
```

Comment:
Whilst the proposed change is more correct in terms of describing the data, I think the intent of the second clause in the original sentence was to highlight that Microsoft and Siemens are separate legal entities so I suggest instead linking to the description of `Organization.id` which explains how that field relates to organisation identifiers.
 
```suggestion
Both Siemens and Microsoft are listed as suppliers on the contract award in OCDS, with `id`s constructed according to [`Organization.id`](../../schema/reference.md#release-schema.json,/definitions/Organization,id).
```